### ISSUE TITLE
Detect php:// as PHP injection

### ISSUE THREAD

**Issue by lifeforms** [SpiderLabs/owasp-modsecurity-crs#301]
`php://` can be used to refer to standard input/POST body, for instance to locate a payload, or do LFI or RFI.

I've found this string in an attack request: `php://filter/resource=./../../../wp-config.php`

Another example from the documentation: `php://filter/resource=http://www.example.com`

It won't hurt to add the following entries to `php-variables.data` in order to detect this coding style being used.

```
php://fd
php://filter
php://input
php://stdin
php://memory
php://temp
```

(Maybe we could also just check on `php://` in a paranoid sibling. This might catch newer additions or ones we missed)

**Comment by lifeforms** [comment]
A pull request is in #315 and ready for review.

I originally intended to add some strings to `php-variables.data`, but I had problems getting the @pm operator to work with the double slashes substring, so I created a separate rule 933140 for it using a regexp. PHP accepts these strings in MiXeD CaSe as well, so the regexp is case-insensitive.

I'm actually pretty happy with this becoming a separate rule, because it now raises the score independently from the existing PHP variable rule.

These strings are a giveaway for some kind of PHP code or a disguised LFI/RFI attempt. Therefore I've put the rule at critical severity.

I also considered just blocking on the string `php://` to be future-proof against new stream additions in the PHP runtime. For me that would be acceptable. But to prevent false matched I decided to keep the regexp conservative for this first version.
